tests/drivers/gpio/gpio_basic_api/boards/it515xx_evb.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2025 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	resources {
+		compatible = "test-gpio-basic-api";
+		out-gpios = <&gpioe 2 0>; /* GPIOE.2 */
+		in-gpios = <&gpioe 3 0>;  /* GPIOE.3 */
```

Comment:
If we select pins that do not have alternate function we won't need to manually change jumpers between doing gpio and i3c testing. (GPIOE2/3 pins also have I3C alternative function)
Can we select pins that have no alternate function to do this test?

Reply:
Sure, we've chosen GPE4 as the input pin, which doesn't have any alternate functions, and GPA3 as the output pin, which isn't used by any other tests.